src/api/app/models/package_issue.rb

```diff
@@ -8,22 +8,12 @@ def self.sync_relations(package, issues)
     retries = 10
     begin
       PackageIssue.transaction do
-        allissues = []
-        issues.map { |h| allissues += h.last }
-
-        # drop not anymore existing relations
-        PackageIssue.where('package_id = ? AND NOT issue_id IN (?)', package, allissues).lock(true).delete_all
-
-        # create missing in an efficient way
-        sql = ApplicationRecord.connection
-        (allissues - package.issues.to_ary).each do |i|
-          sql.execute("INSERT INTO `package_issues` (`package_id`, `issue_id`) VALUES (#{package.id},#{i.id})")
```

Comment:
Looks like we don't recreate the issues anymore, is that right?

Reply:
> Looks like we don't recreate the issues anymore, is that right?

We don't recreate records in the `issues` table... we recreate records in the `package_issues` table...

Reply:
Yeah, well, the point is that I don't see the code for removing them, I only see the code to create them

Reply:
I see [here](https://guides.rubyonrails.org/v7.0/association_basics.html#methods-added-by-has-many-collection-singular-ids-ids) that the `_ids` writer recreates the association :+1: